app/app/components/modals/Wallet/WalletList.tsx

```diff
@@ -45,7 +45,7 @@ const WalletList = ({
       wallets={
         undetected.sort((a, b) => a.id.localeCompare(b.id)) as WalletType[]
       }
-      isWalletValidForChain={isWalletValidForChain}
+      isWalletValidForChain={() => true}
```

Comment:
is this intentional, if so we can probably remove the `isWalletValidForChain` property or set it to default: true.